### PR TITLE
Add unit tests for extensionRegistry provider methods

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Kick off getting this module caught up with unit tests since we have a couple feature requests & need to ensure stability/avoid regressions.

### CODE REVIEW

**Comment by benjaminapetersen** [comment]
Interested in getting some unit test coverage in so that if we merge #64 in order to address #9 for the online team we can avoid regression risk.

File: test/unit/spec/services/extensionRegistry.spec.js [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+      });
+
+      describe('when multiple endpoints are registered', () => {
+        // TODO: verify registered data does not leak into other endpoints
+        it('should do...');
+      });
+
+      // NOTE: depends on PR #64
+      describe('when data items with a weight property are registered', () => {
+        it('should order items by weight', () => {
```
Comment (jwforres) [inline test/unit/spec/services/extensionRegistry.spec.js]: can we xit both of these like you did with the first one

Comment (benjaminapetersen) [reply test/unit/spec/services/extensionRegistry.spec.js]: yup definitely

**Comment by benjaminapetersen** [comment]
@jwforres good to go?

**Review by jwforres — APPROVED** [review]

**Pull request merged**